test/test_grid_addboundaryset.jl

```diff
@@ -200,4 +235,25 @@
         addboundaryfacetset!(grid, topology, "test_boundary_facetset", filter_function)
         @test getfacetset(grid, "test_boundary_facetset") == Ferrite.create_boundaryfacetset(grid, topology, filter_function)
     end
+
+    @testset "mixed grid 3D" begin
+        nodes = reshape([Node(Vec(x, y, z)) for x in -1:1, y in -1:1, z in 0:1], :)
+        for (shape1, shape2) in ((Hexahedron, Wedge), (Wedge, Hexahedron))
+            grid = Grid([generate_cell1(shape1), generate_cell2(shape2)], nodes)
+            topology = ExclusiveTopology(grid)
+            addboundaryfacetset!(grid, topology, "boundary", _ -> true)
+            @test getfacetset(grid, "boundary") == boundary_facets(shape1, shape2)
```

Comment:
Is there a reason why we need to do this both combinations in 3D? Seems like explicitly creating the two 3D cells like for 2D would be cleaner than relying on the `generate_cell1` and `generate_cell2` methods, as well as the `boundary_facets`?